src/containers/Proposal/User/Submitted.jsx

```diff
@@ -20,8 +20,8 @@ const Proposals = (props) => {
   const [hasMoreToLoad, setHasMore] = useState(false);
 
   const { userID } = props;
-  const { proposals, loading, numOfUserProposals, onFetchUserProposals } =
-    useUserProposals(userID);
+  const { proposals, loading, numOfUserProposals, onFetchMoreProposals } =
+    useUserProposals({ userID });
```

Comment:
Code formatting stuff. I think you should use this approach here
```suggestion
  const {
    proposals,
    loading,
    numOfUserProposals,
    onFetchMoreProposals
  } = useUserProposals({ userID });
```

Reply:
Let's let prettify take care of this.